yarn.lock

```diff
@@ -3570,7 +3570,7 @@ command-join@^2.0.0:
   dependencies:
     "@improved/node" "^1.0.0"
 
-command-line-args@^5.0.0:
+command-line-args@^5.0.0, command-line-args@^5.1.1:
```

Comment:
If you end up changing the `package.json` based on my comment above, re-run `yarn` to regenerate this file.

Reply:
I think I did this right but not 100% sure. Quite a large diff but that could be expected from full regenerate? I copied yarn.lock from idyll master and then ran yarn install